### PR TITLE
History modal

### PR DESCRIPTION
**Pull request by alekhrycaiko**
This is the MVP for the modal feature. We're going to iterate on this to make it prettier but all the main components should be functional. 

I'm keeping testing light here since I think this'll be pretty time consuming, I'd rather us have this feature ready to demo and worry about modal testing a bit later. This ticket was already pretty time consuming.

In addition, I've modified the data.sql script in order to accept REQU, PEND, DENY and APPR codes for requests as I think they're more descriptive of what's actually going on (rather than a number).
Backend please review those changes.

An IE11 check still needs to be performed on this ticket.

### CODE REVIEW

File: scripts/js/history_modal.jsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                                <div className="css-body-modal">
+                                    <h4> Products in this order: </h4>
+                                    <ul className="css-modals-productlist">
+                                        {this.getProducts()}
+                                    </ul>
+                                    <div className="css-modal-details">
+                                        <h4> Notes</h4>
+                                        <p> {this.state.notes} </p>
+                                        <h4> Submitted By</h4>
+                                        <p> {this.state.submittedBy}</p>
```
Comment (mandyfung) [inline scripts/js/history_modal.jsx]: Does the backend only return the ID of the employees? Not an issue right now, but something we should probably fix in the future.

Comment (csemiao) [inline scripts/js/history_modal.jsx]: The backend only returns the employee id when looking at the request.  Something we reasoned was that in a large organization with >2000 members, there's really no need to refer to anyone by name.

File: scripts/js/history_modal.jsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            content: {
+                "max-width": "500px",
+                "margin": "0 auto"
+            }
+        }
+
+
+        if (this.state.loading === false && this.state.alert === false) {
+            return(    <Modal
+                    style={customStyles}
```
Comment (mandyfung) [inline scripts/js/history_modal.jsx]: This was what was causing it to fail on IE11, removing this line seemed to have worked

File: scripts/js/history_modal.jsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                    </Modal.Footer>
+                </Modal>
+            );
+        }
+        else if (this.state.loading === false && this.state.alert === true){
+            return(
+                <div>
+                    <Alert title={this.state.alertMessage}></Alert>
+                    <Modal
+                        style={customStyles}
```
Comment (mandyfung) [inline scripts/js/history_modal.jsx]: Same with this line

Comment (alekhrycaiko) [inline scripts/js/history_modal.jsx]: great catch! updated!

**Pull request closed without merging**